templates/dependabot.yml.erb

```diff
@@ -8,11 +8,13 @@ updates:
     schedule:
       interval: "daily"
   # Watch dependencies for current stable only
-<% available_gemfile_paths = Dir.glob("docker-image/**/*/Gemfile") %>
+<% latest_version = Dir.glob("docker-image/*").sort.last %>
```

Comment:
`sort` is inappropriate, `v1.10` will be placed former than `v1.9`.

Reply:
Ah, it should be fixed.